lib/xcode.js

```diff
@@ -144,6 +144,31 @@ const simulatorDevicePairCompatibility = {
 			'5.x': true,        // watchOS 5.x
 			'6.x': true         // watchOS 6.x
 		}
+	},
+	'12.x': {              // Xcode 12.x
+		'10.x': {          // iOS 10.x
```

Comment:
Xcode 12 does not appear to support anything older than iOS 10.3, so this should be `'>=10.3 <11'`.